spec.bs

```diff
@@ -2866,35 +2865,29 @@ a [=list=] of [=interest groups=] |bidIgs|, and a [=list=] of [=bid debug report
    : [=leading bid info/top score=]
    :: |response|'s [=server auction response/score=].

```

Comment:
[=server auction response/score=] can be null. Need to set this to 0.00001 when it's null?
https://crsrc.org/c/content/browser/interest_group/interest_group_auction.cc;drc=451093bbaf7fe812bf67d27d760f3bb64c92830b;l=5851